cluster-setup/ci-upgrade-test-cluster/performance/performance_profile.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: performance.openshift.io/v1alpha1
+apiVersion: performance.openshift.io/v1
```

Comment:
makes sense, but can we have a bit more context in the commit message (few lines) like
1. what's the error we see
2. (thus) why this patch fixes it
3. (if possible) when the issue started to happened
?

Reply:
I also want to see the error. PAO should be able to work with v1alpha1 profiles.

Reply:
@fromanirh @marcel-apf I updated the commit and description message, @marcel-apf please correct me if I wrong.

Reply:
@cynepco3hahue as we discussed offline the conversion webhook is available as soon as the PAO pod is up and running.
We should test it before creating old profiles. This patch will work, hiding the mentioned issue

Reply:
The PR subject remains incorrect

Reply:
Yep I tested it locally and it worked for me, I will close the PR, because it does not really fix anyhting